src/main/utils/resource.ts

```diff
@@ -1,112 +1,112 @@
-/*************************************************************
- * The following logic is part of the CVE applicability scan.*
- * It will be hidden until it is officially released.        *
- * ***********************************************************
- */
-// import * as fs from 'fs';
-// import * as path from 'path';
-// import * as crypto from 'crypto';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as crypto from 'crypto';
 
-// import { JfrogClient } from 'jfrog-client-js';
-// import { IChecksumResult } from 'jfrog-client-js';
-// import { ConnectionUtils } from '../connect/connectionUtils';
-// import { LogManager } from '../log/logManager';
-// import { ConnectionManager } from '../connect/connectionManager';
+import { JfrogClient } from 'jfrog-client-js';
+import { IChecksumResult } from 'jfrog-client-js';
+import { ConnectionUtils } from '../connect/connectionUtils';
+import { LogManager } from '../log/logManager';
 
-// export class Resource {
-//     private static readonly MILLISECONDS_IN_HOUR: number = 3600000;
-//     private TMP_DOWNLOAD_PATH: string = '';
-//     private resourcePath: string = '';
-//     private readonly _defaultJfrogClient: JfrogClient = ConnectionUtils.createJfrogClient(
-//         'https://releases.jfrog.io',
-//         'https://releases.jfrog.io/artifactory',
-//         '',
-//         '',
-//         '',
-//         ''
-//     );
+// Resource classes represent generic instances of external artifact (e.g. binary).
+export class Resource {
+    // You should not change this variable. This may only be done for testing purposes.
+    public static MILLISECONDS_IN_HOUR: number = 3600000;
```

Comment:
Please make sure to do that in all contants
```suggestion
    public static readonly MILLISECONDS_IN_HOUR: number = 3600000;
```

Reply:
This should be overridden for test purposes. The readonly key won't let me set it.